### ISSUE TITLE
How to identify license terms for code in blog articles?

### ISSUE THREAD

**Issue by stephenkilbaneadi** [MicrosoftDocs/feedback#1754]
In this blog article: https://blogs.msdn.microsoft.com/twistylittlepassagesallalike/2011/04/23/everyone-quotes-command-line-arguments-the-wrong-way/

The author provides code, and says, "The function below properly quotes an argument; translate it into your language and coding style of choice." However, that's insufficiently specific to pass a license-compliance review. How do I determine the license terms for this particular example?

More generally, it would be useful if there were an overriding statement of terms, as is done by (for example) Stack Overflow.

Thanks,

steve

**Comment by ryanmajidi** [comment]
@EdisonDa Can you please take a look at this issue?

**Comment by stephenkilbaneadi** [comment]
Did anything ever happen for this?

**Comment by stephenkilbaneadi** [comment]
Still nothing, @EdisonDa ? Has there been no effort to include a license statement site Terms of Use?

**Comment by EdisonDa** [comment]
@ryanmajidi not sure if @kexugit can help here.

**Comment by kexugit** [comment]
@stephenkilbaneadi This is archived content. We do not update content unless it's a critical issue. Also the original author is not working for Microsoft anymore, so it's hard to have the content correctly updated.

**Comment by stephenkilbaneadi** [comment]
Fair enough. Would it be possible to state here in the comments what the terms of use would be, then, please?

**Comment by kexugit** [comment]
@stephenkilbaneadi there's a link to the terms of use in the footer, is that what you were looking for?

**Comment by stephenkilbaneadi** [comment]
@kexugit My recollection is that those terms of use described using the site, but did not cover the terms of use for incorporating code examples on the site into other applications. By analogy, Stack Overflow's terms of use include generic terms for using code examples in applications, if no other more explicit license is stated.

**Comment by kexugit** [comment]
@stephenkilbaneadi There's no other specific terms of uses defined for archived content. The general terms of uses cover the entire Docs site (https://docs.microsoft.com), which should include the archived content.
